src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -21,14 +21,14 @@
   </metadata>
   <files>
     <!-- The subfolder layout for the different TargetFrameworks is intentionally inconsistent -->
-    <!-- net472 and netcoreapp3.1 pack to tools\TargetFramework\* for backcompat -->
-    <file src="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\Sarif.Multitool.exe.config"
-          target="tools\net472" />
-    <file src="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\**"
-          target="tools\net472"
-          exclude="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\Sarif*;bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\WorkItems.dll;" />
-    <file src="bld\bin\Signing\net472\**"
-          target="tools\net472" />
+    <!-- net462 and netcoreapp3.1 pack to tools\TargetFramework\* for backcompat -->
+    <file src="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net462\Sarif.Multitool.exe.config"
```

Comment:
>\net462\ [](http://example.com/codeflow?start=73&length=8)

I tried get the branch, cleaning everything, build, this is the folder I have:
sarif-sdk\bld\bin\AnyCPU_Release\Sarif.Multitool\net472
There is no net462 folder.
Some other changes needed to generate to the net462 instead?